documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,19 +51,34 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+  labels:
+    createdForProviderType: <provider_type> <2>
 type: Opaque
 stringData:
-  user: <user> <1>
-  password: <password> <2>
-  cacert: | <3>
-    <engine_ca_certificate>
-  thumbprint: <vcenter_fingerprint> <4>
-EOF
+  user: <user> <3>
+  password: <password> <4>
+  domainName: <domain_name> <5>
+  projectName: <project_name> <6>
+  regionName: <region name> <7>
+  cacert: | <8>
+    <ca_certificate>
+  url: <api_end_point> <9>
+  thumbprint: <vcenter_fingerprint> <10>
 ----
-<1> Specify the vCenter user or the {rhv-short} {manager} user.
-<2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
-<4> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> The `ownerReferences` section of the manifest is for {osp} only.
+<2>  Specify the type of source provider. Allowed values are `ovirt`, `vsphere`, and `openstack`. This label is needed to verify the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified.
+<3> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
+<4> Specify the user password.
+<5> {osp} only: Specify the domain name.
+<6> {osp} only: Specify the project name.
+<7> {osp} only: Specify the name of the {osp} region.
+<8> {rhv-short} only: {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
+<9> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere, `https://<engine_host>/ovirt-engine/api/` for {rhv-short}, or `https://rhos-d.infra.prod.upshift.rdu2.redhat.com:13000/v3` for {osp}.
```

Comment:
for vsphere and rhv we don't specify concrete URLs whereas we specify one for openstack, how about 
```suggestion
<9> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere, `https://<engine_host>/ovirt-engine/api/` for {rhv-short}, or `https://<identify_service>/v3` for {osp}.
```